src/globus_sdk/services/transfer/client.py

```diff
@@ -112,44 +113,76 @@ class TransferClient(client.BaseClient):
     scopes = TransferScopes
     default_scope_requirements = [Scope(TransferScopes.all)]
 
-    def add_app_data_access_scope(self, collection_id: UUIDLike) -> TransferClient:
+    def add_app_data_access_scope(
+        self, collection_id: UUIDLike | t.Iterable[UUIDLike]
+    ) -> TransferClient:
         """
-        Add a dependent ``data_access`` scope for a given ``collection_id`` to this
-        client's ``GlobusApp``. Useful for resolving ``ConsentRequired`` errors
+        Add a dependent ``data_access`` scope for one or more given ``collection_id``
+        to this client's ``GlobusApp``. Useful for resolving ``ConsentRequired`` errors
         when using standard Globus Connect Server mapped collections.
-        ``collection_id`` must be for a standard Globus Connect Server mapped
-        collection or app driven authentication flows will hit unknown scope errors.
+
+        ``collection_id`` must be for standard Globus Connect Server mapped
```

Comment:
"must _be for_"...perhaps "must _refer to_ GCS mapped collections"?

Also, what is meant by "_standard_ GCS mapped collections"? Are there mapped collections that are not standard?

Reply:
"Standard" was used, I believe, in contrast with "High-Assurance". I think that _some_ docs use "standard" as the antonym for HA, but it's hardly consistent enough that I'd argue in favor of it.

I would tend to put this note into a warning, since it's a big caveat, which gives us more visual room and latitude to explain. So... does the following look right?
```
.. warning::

    This method must only be used on ``collection_ids`` for non-High-Assurance GCS Mapped Collections.
    Use on other collection types, e.g., on GCP Mapped Collections or any form of Guest Collection,
    will result in "Unknown Scope" errors during the login flow.
```